.github/workflows/ubuntu_build.yml

```
@@ -0,0 +1,66 @@
+name: Calculix 2.17 with preCICE adapter test build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, ubuntu-20.04]
+        include:
+          - os: ubuntu-18.04
+            name: bionic
+          - os: ubuntu-20.04
+            name: focal
+#          - os: ubuntu-21.04
+#            name: hirsute
+    runs-on: ${{matrix.os}}
+    continue-on-error: true
+
+    steps:
+    - name: Update system
+      run: sudo apt update && sudo apt upgrade -y
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.ref }}
+    - name: get preCICE
+      run: wget https://github.com/precice/precice/releases/download/v2.3.0/libprecice2_2.3.0_${{matrix.name}}.deb &&
+        sudo apt install ./libprecice2_2.3.0_${{matrix.name}}.deb -y
+    - name: install dependencies
+      run: sudo apt install libarpack2-dev libspooles-dev libyaml-cpp-dev -y
+    - name: Download Calculix
+      run: wget http://www.dhondt.de/ccx_2.17.src.tar.bz2
+    - name: Unpack Calculix
+      run: bzip2 -d ccx_2.17.src.tar.bz2 && tar -xf ccx_2.17.src.tar
```

Why not directly
```suggestion
      run: tar -xjf ccx_2.17.src.tar.bz2
```
?

---

Never heard of it, good to know !

---

:grimacing: https://precice.org/adapter-calculix-get-calculix.html#get-the-source